app.js

```diff
@@ -23,11 +23,27 @@ const CookieParser = require('cookie-parser')
 const DrainManager = require('./app/js/DrainManager')
 const HealthCheckManager = require('./app/js/HealthCheckManager')
 
+// NOTE: debug is invoked for every blob that is put on the wire
+const ENABLED_LOG_LEVELS = ['error', 'warn']
+const LOG_LEVELS = ['error', 'warn', 'info', 'debug', 'log']
+const socketIoLogger = LOG_LEVELS.reduce((fakeLogger, level) => {
+  if (ENABLED_LOG_LEVELS.includes(level)) {
+    fakeLogger[level] = (...message) => {
+      logger.info({ fromSocketIo: true, originalLevel: level }, ...message)
+    }
+  } else {
+    fakeLogger[level] = function () {}
+  }
+  return fakeLogger
+}, {})
+
```

Comment:
Since there are only 2 actual loggers and 3 empty ones, maybe it's simpler just to define them explicitly?

```javascript
const socketIoLogger = {
 'error': (...message) => { logger.info({fromSocketIo:true, originalLevel: 'error'}, ...message)}
 'warn': (...message) => { logger.info({fromSocketIo:true, originalLevel: 'warn'}, ...message)}
 'info': function () {}
 ...
}
```

Reply:
Done and force-pushed: https://github.com/overleaf/real-time/pull/190/commits/cd63441f1e50b48fef5b45dac5fc833953b4a5c8